analytics_data_api/management/commands/create_elasticsearch_learners_indices.py

```diff
@@ -0,0 +1,84 @@
+from elasticsearch import Elasticsearch
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Creates ElasticSearch indices used by the Analytics Data API'
+
+    def handle(self, *args, **options):
+        es = Elasticsearch([settings.ELASTICSEARCH_LEARNERS_HOST])
+        if not es.indices.exists(settings.ELASTICSEARCH_LEARNERS_INDEX):
```

Comment:
Nit: It might be nice to know if an index is already created and this results in a no-op.